docs/source/bond_family_guide/sysadmin_guide/configuration.rst

```diff
@@ -0,0 +1,225 @@
+..
+   INTEL CONFIDENTIAL
+   Copyright(c) 2016 Intel Corporation All Rights Reserved.
+
+   The source code contained or described herein and all documents related to
+   the source code ("Material") are owned by Intel Corporation or its suppliers
+   or licensors. Title to the Material remains with Intel Corporation or its
+   suppliers and licensors. The Material contains trade secrets and proprietary
+   and confidential information of Intel or its suppliers and licensors. The
+   Material is protected by worldwide copyright and trade secret laws and
+   treaty provisions. No part of the Material may be used, copied, reproduced,
+   modified, published, uploaded, posted, transmitted, distributed, or
+   disclosed in any way without Intel's prior express written permission.
+
+   No license under any patent, copyright, trade secret or other intellectual
+   property right is granted to or conferred upon you by disclosure or delivery
+   of the Materials, either expressly, by implication, inducement, estoppel or
+   otherwise. Any license under such intellectual property rights must be
+   express and approved by Intel(R) in writing.
+
+*************
+Configuration
+*************
+
+Overview
+========
+
+This article guides you through the configuration and startup of the Sawtooth
+Lake Bond Family validator service that you installed in :doc:`installation`.
+
+
+Prerequisites
+=============
+
+Networking
+----------
+
+1. Your node must be able to communicate with other nodes in the network, as
+covered in the networking section of :doc:`installation`.
+
+2. You need information in the form of an email document from your Intel
```

Comment:
True. @rbuysse  or @aludvik : how will be be providing this info to people? Also, have the field's required changed at all? Thanks!